jwst/pipeline/resample.cfg

```diff
@@ -1,11 +1,2 @@
 name = "resample"
 class = "jwst.resample.ResampleStep"
-
-single = False
```

Comment:
Are all of these set in the step `spec` so that we still get the same default values?

Reply:
Yes. This is actually preliminary work for the epic that deals with deprecating out the cfg specifications.